freshmaker/pyxis_gql.py

```diff
@@ -47,10 +48,9 @@ def connect(self):
 class PyxisGQL:
     def __init__(self, url, cert):
```

Comment:
Good point!

I removed the 'cert' parameter from PyxisGQL and cascaded this change to the rest of the code. I had to make a minor change to the ContainerAPI class in the container.py module, since it received a parameter that was just forwarded as 'cert'.

I also updated the tests for the container and pyxis_gql modules.